docs/webpack.config.base.js

```diff
@@ -1,5 +1,5 @@
-var path = require('path'); // eslint-disable-line no-var
-var ExtractTextPlugin = require('extract-text-webpack-plugin'); // eslint-disable-line no-var
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
```

Comment:
we don't need extract-text-webpack-plugin ?

Reply:
Nop! That is used to extract css from code into .css files. `mini-css-extract-plugin` is the new plugin for it. `extract-text-webpack-plugin` is "deprecated"